worlds/lingo/test/TestPilgrimage.py

```diff
@@ -88,6 +88,35 @@ def test_access(self):
         self.assertTrue(self.can_reach_location("Pilgrim Antechamber - PILGRIM"))
 
 
+class TestPilgrimageRequireStartingRoom(LingoTestBase):
+    options = {
+        "enable_pilgrimage": "true",
+        "shuffle_colors": "false",
+        "shuffle_doors": "complex",
+        "pilgrimage_allows_roof_access": "false",
+        "pilgrimage_allows_paintings": "false",
+        "early_color_hallways": "false"
+    }
+
+    def test_access(self):
+        doors = ["Second Room - Exit Door", "Crossroads - Roof Access", "Hub Room - Crossroads Entrance",
+                 "Outside The Undeterred - Green Painting", "Outside The Undeterred - Number Hunt",
+                 "Starting Room - Street Painting", "Outside The Initiated - Shortcut to Hub Room",
+                 "Directional Gallery - Shortcut to The Undeterred", "Orange Tower First Floor - Salt Pepper Door",
+                 "Color Hunt - Shortcut to The Steady", "The Bearer - Entrance",
+                 "Orange Tower Fifth Floor - Quadruple Intersection", "The Tenacious - Shortcut to Hub Room",
+                 "Outside The Agreeable - Tenacious Entrance", "Crossroads - Tower Entrance",
+                 "Orange Tower Fourth Floor - Hot Crusts Door", "Challenge Room - Welcome Door",
+                 "Number Hunt - Challenge Entrance", "Welcome Back Area - Shortcut to Starting Room"]
+
+        for door in doors:
+            print(door)
```

Comment:
While maybe nice when debugging the test, that print should probably not be here. If you want to get a proper message (on failure or verbose mode), please use the 2nd argument to `self.assertFalse()` or put the assert into an `with self.subTest(door=door):`

Reply:
Whoops, that was mistakenly left in from debugging. Fixed.